### PR TITLE
Update to GNOME 40

### PR DESCRIPTION
**Pull request by Eonfge**
@bilelmoussaoui Seeing how 3.38 also never got merged, could you review this one?

### CODE REVIEW

File: com.github.fabiocolacio.marker.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
         "sources": [
           {
             "type": "archive",
-            "url": "https://www.abisource.com/downloads/enchant/1.6.0/enchant-1.6.0.tar.gz",
+            "url": "http://www.abisource.com/downloads/enchant/1.6.0/enchant-1.6.0.tar.gz",
```
Comment (bilelmoussaoui) [inline com.github.fabiocolacio.marker.json:30]: Why the change?

Comment (Eonfge) [reply com.github.fabiocolacio.marker.json:30]: they didn't update their certificate, Flathub won't accept it so I figured I might just as well use the non-secure version.

Comment (Eonfge) [reply com.github.fabiocolacio.marker.json:30]: It's a 10 year old package... I don't expect them to update it any time soon.

**Comment by Eonfge** [comment]
bot, build

File: com.github.fabiocolacio.marker.json (hunk 1 above)
Comment (hfiguiere) [reply com.github.fabiocolacio.marker.json:30]: there are new issue with letsencrypt certificates I need to fix.

Try 1.6.1 from github:

https://github.com/AbiWord/enchant/releases/tag/enchant-1-6-1

Comment (Eonfge) [reply com.github.fabiocolacio.marker.json]: Thanks for the update. 1.6.1 seems to work fine so I'll use that version instead. Perhaps also change the info on the website, because I was unaware that a newer version was hosted on a different site.

Comment (hfiguiere) [reply com.github.fabiocolacio.marker.json]: Like this?

image

**Comment by hfiguiere** [comment]
This should close #27 and close #29

File: com.github.fabiocolacio.marker.json (hunk 1 above)
Comment (hfiguiere) [reply com.github.fabiocolacio.marker.json]: FWIW the cert problem was fixed. But github being the official stuff, it's no longer relevant.
